content/automate/ManageIQ/Transformation/Common.class/__methods__/vmchecktransformed.rb

```diff
@@ -30,7 +30,7 @@ def update_total_percentage
           end
 
           def main
-            @task.get_conversion_state
+            # @task.get_conversion_state is now done from InfraConversionJob
```

Comment:
@jameswnl Minor, but is there any reason to maintain this commented line?  I do not think it adds much value to retain it.